docs/src/cli/delegate-stake.md

```diff
@@ -0,0 +1,136 @@
+# Earn Staking Rewards
+
+After you have [recieved SOL](transfer-tokens.md), you might consider putting
+it to use by delegating stake to a validator node. Solana is a Proof of Stake
+blockchain, which means that holding tokens gives you the option to influence
+which validator nodes participate in securing the ledger. The more tokens you
+hold, the more influence you have. The running assumption of any Proof of Stake
+blockchain is that the more tokens you hold, the more you have to lose if a
+validator node acts maliciously.
+
+As a tokenholder, it is your option to use your tokens as *stake*. Stake is
+what we call tokens when they reside in a *stake account*. From the stake
+account, you can delegate that stake to validators. Solana weights validator
+votes by the amount of stake delegated to them, which gives those validators
+more influence in determining then next valid block of transactions in the
+blockchain. Solana then generates new SOL periodically to reward stakers and
+validators. You earn more rewards the more stake you delegate.
+
+## Create a Stake Account
+
+To delegate stake, you will need to move your tokens into a stake account.
+All commands in this section assume you have
+[set a default fee-payer](transfer-tokens.md#set-a-default-fee-payer).
+
+Create a stake account with the `solana create-stake-account` command:
+
+```bash
+solana create-stake-account --from=<KEYPAIR> <ACCOUNT_KEYPAIR> <AMOUNT>
+```
+
+`<AMOUNT>` tokens are transferred from the account at `<KEYPAIR>` to a new
+stake account at the public key of `<ACCOUNT_KEYPAIR>`.  Record the account
```

Comment:
Do we need to hand-hold through or at least reference the need to create this new keypair?

Reply:
+1 for adding a step to create the new keypair.